DCEC.py

```diff
@@ -239,6 +243,7 @@ def fit(self, x, y=None, batch_size=256, maxiter=2e4, tol=1e-3,
 
     # load dataset
     from datasets import load_mnist, load_usps, load_fasta
+    nClusters = args.n_clusters
```

Comment:
Same here n_clusters, Also I think that the variable is dead and can be removed.